src/NClient/NClient.Standalone/ClientProxy/Generation/Interceptors/ClientInterceptor.cs

```diff
@@ -148,32 +153,58 @@ internal class ClientInterceptor<TClient, TRequest, TResponse> : AsyncIntercepto
             }
         }
         
-        private async Task<object?> ExecuteHttpResponseAsync(ITransportNClient<TRequest, TResponse> transportNClient, IRequest request, Type resultType, IResiliencePolicy<TRequest, TResponse>? resiliencePolicy, CancellationToken cancellationToken)
+        private async Task<object?> ExecuteHttpResponseAsync(ITransportNClient<TRequest, TResponse> transportNClient, IRequest request, Type resultType, IResiliencePolicy<TRequest, TResponse>? resiliencePolicy, ICachingAttribute? cachingAttribute = default, CancellationToken cancellationToken = default)
         {
+            if (await _responseCacheWorker.TryGet(request, cancellationToken) is { } cachedResult)
```

Comment:
What if the cache becomes unavailable and throws exceptions? I think the client should continue to work without a cache. The same goes for saving the cache

Reply:
Failfast, no?